recirq/otoc/loschmidt/tilted_square_lattice/tilted_square_lattice.py

```diff
@@ -108,7 +108,10 @@ def _json_namespace_(cls):
         return 'recirq.otoc'
 
     def _json_dict_(self):
-        return dataclass_json_dict(self, namespace=self._json_namespace_())
+        if cirq.__version__ < '0.15':
+            return dataclass_json_dict(self, namespace=self._json_namespace_())
```

Comment:
is this necessary? I think the `_json_namespace_` was already present in 0.14 and the shim was for older compatibility

Reply:
I'll keep it for now to support 0.13 compatibility.  Feel free to remove it after you upgrade the requirements.